bazel-java-deps.bzl

```diff
@@ -197,6 +198,7 @@ def install_java_deps():
             "org.xerial:sqlite-jdbc:3.36.0.1",
             "com.fasterxml.jackson.core:jackson-core:2.12.0",
             "com.fasterxml.jackson.core:jackson-databind:2.12.0",
+            "org.scala-lang:scala-library:{}".format(scala_version),
```

Comment:
Is the intent here to pin the Scala version? How does this work?

Reply:
It pins the scala std library version because the in-place mechanisms we have are not enough to pin the version of the scala std library. Specifically this causes that all libraries which depend on the std lib will use the same version of the std lib because a speciifc version of the std lib is already in our must-have dependency list (at least it seems to work like this).